components/validation/demo/validation.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+	<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+	<script type="module">
+			import '../../inputs/input-checkbox.js';
+			import '../../inputs/input-text.js';
+			import '../../typography/typography.js';
+			import '../../button/button.js';
+			import '../../colors/colors.js';
+	</script>
+	<style>
+		body {
+			padding: 30px;
+		}
+	</style>
+</head>
+
+<body>
+
+	<form action="/action" onsubmit="return validate()">
```

Comment:
I think it would be worthwhile for the PoC to wrap this up in a mock app component, so all this ends up being contained in another shadowDOM.